Geometry/HcalCommonData/src/HcalNumberingFromDDD.cc

```diff
@@ -5,11 +5,10 @@
 
 #include "CondFormats/GeometryObjects/interface/HcalParameters.h"
 #include "Geometry/HcalCommonData/interface/HcalNumberingFromDDD.h"
+#include "DetectorDescription/DDCMS/interface/DDUnits.h"
 
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 
-#include "CLHEP/Units/GlobalPhysicalConstants.h"
-#include "CLHEP/Units/GlobalSystemOfUnits.h"
 #include <iostream>
 
 //#define EDM_ML_DEBUG
```

Comment:
```suggestion
using namespace dd;
using namespace dd::operators;

//#define EDM_ML_DEBUG
```

Reply:
Needed by the Core version of `DDUnits.h`.

Reply:
Done